qiskit_ibm_transpiler/wrappers/function_transpile.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+
+# (C) Copyright 2025 IBM. All Rights Reserved.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""Transpilation via the transpiler qiskit function"""
+
+import logging
+from typing import Dict, List, Union
+
+from qiskit import QuantumCircuit
+from qiskit.transpiler.exceptions import TranspilerError
+from qiskit_serverless import ServerlessClient
+
+from qiskit_ibm_transpiler.types import OptimizationOptions
+
+from .base import _get_token_from_system
+
+# TODO: add actual logging if required, else remove this
+logger = logging.getLogger(__name__)
+
+
+class QiskitTranspilerFunction:
+    """A helper class that used the qiskit runtime function interface for transpilation"""
+
+    SERVERLESS_URL = "https://qiskit-serverless.quantum.ibm.com"
+    DEFAULT_CHANNEL = "ibm_cloud"
```

Comment:
DEFAULT_CHANNEL should be configured as `ibm_quantum_platform` , `ibm_cloud` is deprecated.